install.yaml

```diff
@@ -1,11 +1,28 @@
 name: mongo
 
 pre_install_actions:
-- |
-  #ddev-description:Create config.mongo.yaml with webimage_extra_packages in it
-  printf '#ddev-generated\nwebimage_extra_packages: ["php${DDEV_PHP_VERSION}-mongodb"]\n' >.ddev/config.mongo.yaml
+  - |
+    #ddev-description:Create config.mongo.yaml with webimage_extra_packages in it
+    printf '#ddev-generated\nwebimage_extra_packages: ["php${DDEV_PHP_VERSION}-mongodb"]\n' >.ddev/config.mongo.yaml
+  # Ensure we're on DDEV 1.23+. It's required for the `redis-commander` command (launch by port).
+  - |
+    #ddev-nodisplay
+    #ddev-description:Checking DDEV version
+    (ddev debug capabilities | grep corepack >/dev/null) || (echo "Please upgrade DDEV to v1.23+ to enable launching." && false)
```

Comment:
Rather than this approach, it's easier now to use `ddev_version_constraint` in the config.mongo.yaml if there is one.

Reply:
Sorry, I should have said `ddev_version_constraint` in the install.yaml., https://github.com/ddev/ddev-addon-template/blob/main/install.yaml#L85